internal/nvidia/nvidia.go

```diff
@@ -0,0 +1,130 @@
+package nvidia
+
+import (
+	"fmt"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+	"github.com/markusressel/fan2go/internal/configuration"
+	"github.com/markusressel/fan2go/internal/fans"
+	"github.com/markusressel/fan2go/internal/sensors"
+)
+
+type NvidiaController struct { // TODO: why "controller"?
```

Comment:
not sure what else to call it :man_shrugging: :smile:

Reply:
Device? Chip? IDK :-D

I guess I would've used Controller for an instance that controls all devices of this backend (had to call that nvidiaHandler now) - but of course you couldn't know that something like that would be needed in the future :)

Reply:
I think even when I named HwMonController I didn't really know what to call it.
It doesn't really matter though, use whatever feels right to you.